### PR TITLE
Strong naming consent client

### PR DESCRIPTION
**Pull request by NakulSeerpu**
For better compatibility with .NET Framework public KMD Logic assemblies will be strongly named. The private key will be included in the repo as we're not using it for any security related stuff.

### CODE REVIEW

File: Directory.Build.props [hunk 1]
```diff
@@ -9,10 +9,12 @@
...
+    <VersionPrefix Condition=" '$(VersionPrefix)' == '' ">2.0.0</VersionPrefix>
+    <Version Condition=" '$(VersionSuffix)' != '' ">$(VersionPrefix)-$(VersionSuffix)</Version>
+    <Version Condition=" '$(Version)' == '' ">$(VersionPrefix)</Version>
     <Authors>KMD</Authors>
     <PackageTags>kmd;logic;consent</PackageTags>
-    <PackageIconUrl>https://console.kmdlogic.io/favicon-32x32.png</PackageIconUrl>
+    <PackageIcon>kmd-logic-icon.png</PackageIcon>
     <PackageProjectUrl>https://github.com/kmdlogic/kmd-logic-consent-client</PackageProjectUrl>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <RepositoryUrl>https://github.com/kmdlogic/kmd-logic-consent-client</RepositoryUrl>
```
Comment (Araballi) [inline Directory.Build.props:20]: `<GenerateAssemblyVersionAttribute>true</GenerateAssemblyVersionAttribute>` change is missing

**Review by Araballi — CHANGES_REQUESTED** [review]
Do Build.props files has missing changes

File: Directory.Build.props (hunk 1 above)
Comment (NakulSeerpu) [reply Directory.Build.props:20]: Fixed

**Review by Araballi — APPROVED** [review]
LGTM

**Pull request merged**